packages/ibm-products/src/components/CreateTearsheet/preview-components/MultiStepWithCustomButton.js

```diff
@@ -0,0 +1,103 @@
+/* eslint-disable react/prop-types */
+/**
+ * Copyright IBM Corp. 2021, 2023
+ *
+ * This source code is licensed under the Apache-2.0 license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+import React, { useState } from 'react';
+import { Button } from '@carbon/react';
+import cx from 'classnames';
+import { pkg } from '../../../settings';
+import { CreateTearsheet } from '../CreateTearsheet';
+import { CreateTearsheetStep } from '../CreateTearsheetStep';
+import { SlugSample } from '../../../global/js/story-parts/slug';
+
+const blockClass = `${pkg.prefix}--tearsheet-create-multi-step`;
+
+export const MultiStepWithCustomButton = ({
```

Comment:
after some discussion we decided that we didn't want to highlight this functionality too much. instead of giving it it's own story could you move it to a control similar to the [4 button layout in regular tearsheet](https://carbon-for-ibm-products.netlify.app/?path=/story/ibm-products-components-tearsheet--tearsheet&args=actions:9&globals=viewport:basic)?